src/modules/esl-tab/core/esl-tabs.ts

```diff
@@ -19,7 +19,7 @@ export class ESLTabs extends ESLBaseElement {
   public static is = 'esl-tabs';
 
   /** Marker to enable scrollable mode */
```

Comment:
Docs update with possible values

Reply:
Also, it sounds like we need the default value
because we should support 3 states (disabled, centered, side alignment)
and we should not break the existing API:
scrollable = false (no attribute) -> disabled
scrollable = true (attribute with no value) -> side aligned scrollable

Reply:
Thanks!